src/components/stressComment/stressComment.css

```
@@ -0,0 +1,23 @@
+.StessCommentContainer {
```

Following up on the JS name change.
```suggestion
.StressCommentContainer {
```